shell/platform/embedder/embedder_task_runner.h

```diff
@@ -24,10 +24,12 @@ class EmbedderTaskRunner final : public fml::TaskRunner {
     std::function<bool(void)> runs_task_on_current_thread_callback;
   };
 
-  EmbedderTaskRunner(DispatchTable table);
+  EmbedderTaskRunner(DispatchTable table, size_t embedder_identifier);
 
   ~EmbedderTaskRunner() override;
 
+  size_t GetEmbedderIdentifier() const;
```

Comment:
Might be worth documenting the class and methods

Reply:
Done. Documented all methods and the class.